test/spec/modules/invibesBidAdapter_spec.js

```diff
@@ -1123,22 +1185,29 @@ describe('invibesBidAdapter:', function () {
 
     it('returns an iframe with params if enabled', function () {
       top.window.invibes.optIn = 1;
-      global.document.cookie = 'ivvbks=17639.0,1,2';
       let response = spec.getUserSyncs({iframeEnabled: true});
       expect(response.type).to.equal('iframe');
       expect(response.url).to.include(SYNC_ENDPOINT);
       expect(response.url).to.include('optIn');
-      expect(response.url).to.include('ivvbks');
     });
 
     it('returns an iframe with params including if enabled', function () {
       top.window.invibes.optIn = 1;
-      global.document.cookie = 'ivvbks=17639.0,1,2;ivbsdid={"id":"dvdjkams6nkq","cr":' + Date.now() + ',"hc":0}';
+      global.document.cookie = 'ivbsdid={"id":"dvdjkams6nkq","cr":' + Date.now() + ',"hc":0}';
+      let response = spec.getUserSyncs({iframeEnabled: true});
+      expect(response.type).to.equal('iframe');
+      expect(response.url).to.include(SYNC_ENDPOINT);
+      expect(response.url).to.include('optIn');
+      expect(response.url).to.include('ivbsdid');
+    });
+
+    it('returns an iframe with params including if enabled read from LocalStorage', function () {
+      top.window.invibes.optIn = 1;
```

Comment:
can you confirm you have access to storage in your test ? https://github.com/prebid/Prebid.js/pull/8279

Reply:
yes, I just made a sync with the MASTER from prebid and re-ran all my tests and the passed.

Reply:
right but can you make sure your tests will also pass when #8279 is merged?

Reply:
yes, I've made the necessary updates and tested with the default config changed as in #8279